.github/workflows/nightly-ci.yml

```diff
@@ -0,0 +1,42 @@
+name: Nightly CI
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+    tags: '*'
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Comment:
I've added this, but it doesn't seem to do the job?

Reply:
I'm asking at Slack..